contracts/pods/RoleBastion.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {TribeRoles} from "../core/TribeRoles.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {Core} from "../core/Core.sol";
+
+/// @title RoleBastion
+/// @notice Used by ROLE_ADMIN to create new roles. Granted GOVERNOR role with a simple API
+contract RoleBastion is CoreRef {
+    event BastionRoleCreate(bytes32 indexed role, bytes32 roleAdmin);
+
+    constructor(address _core) CoreRef(_core) {}
+
+    /// @notice Create a role whose admin is the ROLE_ADMIN
+    /// @param role Role to be created
+    /// @dev Function is used by an address with ROLE_ADMIN, e.g. the TribalCouncil, to create
+    ///      non-major roles. A check is made to ensure that the role to be created does not
+    //       have an admin i.e. that it is a new admin
+    function createRole(bytes32 role)
+        external
+        onlyTribeRole(TribeRoles.ROLE_ADMIN)
+    {
+        bytes32 roleAdmin = core().getRoleAdmin(role);
+        require(roleAdmin == bytes32(0), "Role already exists");
```

Comment:
We should not allow a role that is bytes32(0) to be created using this `createRole` method as that is the CONTRACT_ADMIN_ROLE in core.

<img width="731" alt="Screen Shot 2022-03-25 at 11 45 33 AM" src="https://user-images.githubusercontent.com/34463580/160182716-fe0bdfaf-a59b-438e-8a45-5dcd3f60d9a2.png">

Reply:
It is possible to create and grant a role of `bytes32(0)` - it has not yet been created in core. I added an integration test that validates that it can be created. As I understand it, that would allow the TribalCouncil to create a powerful role, which it could then grant. 

Have added a check to prevent creating the `bytes32(0)` role + a test to validate. I'm not sure I follow what you mean by granting another contract admin of 0x0?

Reply:
`    bytes32 public constant DEFAULT_ADMIN_ROLE = 0x00; ` in AccessControl.sol

The 0 role is the default admin of all non-created roles and is super powerful. Good to make sure this can't be granted out